lib/Mojo/IOLoop.pm

```diff
@@ -515,8 +499,8 @@ Accept TCP/IP and UNIX domain socket connections with L<Mojo::IOLoop::Server> an
 L<Mojo::IOLoop::Stream>, takes the same arguments as L<Mojo::IOLoop::Server/"listen">.
 
   # Listen on random port
-  my $id = Mojo::IOLoop->server({address => '127.0.0.1'} => sub {
-    my ($loop, $stream, $id) = @_;
+  my $id = Mojo::IOLoop->server({address => '127.0.0.1'} =>
+    sub ($loop, $stream, $id) {
```

Comment:
This should be collapsible to one line with the new line length limit, as well as some similar changes